doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1585,7 +1585,7 @@ Trusted Firmware-M support
        - --
        - Experimental
        - --
-       - Experimental
+       - Experimental [1]_
```

Comment:
```suggestion
       - Experimental\ :sup:`1`
```

Reply:
We add this a sup not as a footnote. https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/software_maturity.html#wi-fi_feature_support